config/brakeman.ignore

```diff
@@ -0,0 +1,29 @@
+{
+  "ignored_warnings": [
+    {
+      "warning_type": "Mass Assignment",
+      "warning_code": 105,
+      "fingerprint": "491d5b53c0254f09b59ac8334571eea772e9b54c5ebb49202cd2b79392721999",
+      "check_name": "PermitAttributes",
+      "message": "Potentially dangerous key allowed for mass assignment",
+      "file": "app/controllers/users_controller.rb",
+      "line": 35,
+      "link": "https://brakemanscanner.org/docs/warning_types/mass_assignment/",
+      "code": "params.require(:user).permit(:role)",
+      "render_path": null,
+      "location": {
+        "type": "method",
+        "class": "UsersController",
+        "method": "user_params"
+      },
+      "user_input": ":role",
+      "confidence": "Medium",
+      "cwe_id": [
+        915
+      ],
+      "note": ""
+    }
+  ],
+  "updated": "2023-04-11 12:44:26 +0100",
+  "brakeman_version": "5.4.1"
+}
```

Comment:
Is this safe? We want to set user role, we are using strong params, is the check just to catch times when that's not the case?

Reply:
This is a weird one. I wonder if it's flagging it up because it thinks it "auth" related, i.e params on "user" and "roles" so would be unsafe to mass assign which we aren't 

Should be alright to just ignore this as you have done.